sql/updates/latest-dev.sql

```diff
@@ -0,0 +1,91 @@
+-- Recreate _timescaledb_catalog.chunk table --
+CREATE TABLE _timescaledb_catalog.chunk_tmp
+AS SELECT * from _timescaledb_catalog.chunk;
+
+CREATE TABLE tmp_chunk_seq_value AS
+SELECT last_value, is_called FROM _timescaledb_catalog.chunk_id_seq;
```

Comment:
Would using `pg_get_serial_sequence('_timescaledb_catalog.chunk', 'id')` allow us to get the name, or is what I'm saying wrong?